src/patch-events.js

```diff
@@ -275,6 +275,10 @@ export function addEventListener(type, fn, optionsOrCapture) {
           return;
         }
       }
+      if(fn.handleEvent) {
```

Comment:
If a function has the `handleEvent` property added, only the function body is invoked by the event listener.

```js
let handler = typeof fn === 'function' ? fn : fn.handleEvent;
if (handler) {
  return handler.call(this, e);
}
```

Even better, the type of the handler should be determined in `addEventListener`